common/src/main/java/tachyon/worker/WorkerNetAddress.java

```diff
@@ -40,6 +41,18 @@ public WorkerNetAddress(String host, int rpcPort, int dataPort, int webPort) {
   }
 
   /**
+   * Constructs the worker net address from thrift construct.
+   *
+   * @param netAddress the thrift net address
+   */
+  public WorkerNetAddress(NetAddress netAddress) {
```

Comment:
It seems we only use netAddress for workers. Do you think we'll have a use case for master in the future?